integration_tests/features/Mempool.feature

```diff
@@ -169,3 +169,25 @@ Feature: Mempool
     Then SENDER has TX11 in MINED state
     Then SENDER has TX12 in MINED state
     Then SENDER has TX13 in MINED state
+
+    @critical
+    Scenario: Mempool unconfirmed transactions
+      Given I have 1 seed nodes
+      And I have a base node BN1 connected to all seed nodes
+      When I mine a block on BN1 with coinbase CB1
+      When I mine a block on BN1 with coinbase CB2
+      When I mine a block on BN1 with coinbase CB3
+      When I mine a block on BN1 with coinbase CB4
+      When I mine a block on BN1 with coinbase CB5
+      When I mine 5 blocks on BN1
+      When I create a custom fee transaction TX1 spending CB1 to UTX1 with fee 80
+      When I create a custom fee transaction TX2 spending CB1 to UTX1 with fee 80
+      When I create a custom fee transaction TX3 spending CB1 to UTX1 with fee 80
+      When I create a custom fee transaction TX4 spending CB1 to UTX1 with fee 80
+      When I create a custom fee transaction TX5 spending CB1 to UTX1 with fee 80
```

Comment:
Did you mean to spend CB1 five times?

Reply:
oh no, though it actually doesn't matter as we are just looking at the mempool which is why the test passed

Reply:
Yeah I figured, but I guess you don't need to mine 5 coinbases then 🤣

Reply:
Ja, I will clean up as I fix the merge issue currently in dev

Reply:
Nice one